src/classes/STG_SettingsManager_TEST.cls

```diff
@@ -575,12 +596,14 @@ public with sharing class STG_SettingsManager_TEST {
     * @description Tests the Recurring Donations panel
     * @return void
     */
-    public static testMethod void testPanelRD() {
+    @isTest
+    public static void testPanelRD() {
         STG_PanelRD_CTRL panel = new STG_PanelRD_CTRL();
         system.assertEquals('idPanelRD', panel.idPanel());
         panel.init();
         system.assertNotEquals(null, panel.listSORDCloseOptions);
         system.assertNotEquals(null, panel.listSORDCloseOptions);
+        system.assertNotEquals(null, panel.listInstallmentCreationOptions);
```

Comment:
Let's verify select options have correct values in this or a separate unit test.